dashboard/pkg/epinio/edit/applications.vue

```diff
@@ -191,6 +192,13 @@ export default Vue.extend<Data, EpinioCompRecord, EpinioCompRecord, EpinioCompRe
     updateManifestConfigurations(changes: string[]) {
       this.set(this.value.configuration, { configurations: changes });
     },
+    validate(value, tab) {
+      if (tab) {
+        this.tabErrors[tab] = !value;
+      }
+
+      this.steps[0].ready = value;
```

Comment:
The `step` `ready` property determines if the wizard can let the user go `next` to the step (user cannot progress to a step that is not `ready`). 

In the edit context where this is no wizard, the `validation-passed` property in `CruResource` should tie in to the tabErrors values

Reply:
I've moved the validation check to tab errors, I also think it's more appropriate; note that the 'Update Source' in the tab where we use the Wizard saves settings as well, so I think we should still disable the button -  `this.steps[0].ready = false` - in case of tab errors.

Reply:
ah, i see. It's disabling the button in the source tab... which is a wizard step, rather than control the flow of a wizard.

i think i got my statement wrong, looks like ready is specifically for the current step and whether the user can progress away from it.